### DISCUSSION TITLE
How to find dungeon ?

### DISCUSSION THREAD

**Discussion by Tokarif** [MinicraftPlus/minicraft-plus-revived#673] [Q&A] [answered]
I Killed the air wizard on 2 worlds consecutively, I got the message saying i can acces the dungeon... I get to the center of lava level....but staircaise is nowhere to be found. Did I do something wrong ? I have to extend my search around ??

**Comment by BenCheung0422** [comment]
The dungeon gate is neither necessarily located nor fixed at the center of the level.

**Comment by Litorom** [comment, marked as answer]
The gate is randomly generated according to the code. It used to generate at the world center in older versions but that’s not the case anymore.
You may have to extend your search. If you want I can view your world if you zip the world folder and post it here.

**Comment by Tokarif** [comment]
Thank you both :D I'll try to find it by myself... but lava lighting makes my FPS drop next to 0... it will be kinda long :p
